torch/onnx/symbolic_registry.py

```diff
@@ -73,6 +79,8 @@ def register_ops_in_version(domain: str, version: int):
 
 
 def get_ops_in_version(version: int):
+    if not _symbolic_versions:
```

Comment:
nit: this changes to lazy loading `_symbolic_versions`, instead of loading when `symbolic_registry.py` is loaded. This adds assumption that `get_ops_in_version` is the only/first entry point for using `_symbolic_versions`.

Reply:
Done